bin/test

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
+
+main() {
+  python -m pytest
+}
+
+main "$@"
```

Comment:
Just a handy script to run the tests from bash.

You can always just run `python -m pytest` or even just `pytest` from the root of the project